java17-springboot3.2.x/Dockerfile

```
@@ -0,0 +1,44 @@
+FROM  --platform=linux/x86_64 debian:bookworm AS build
+
+RUN set -xe ; \
+    apt -yqq update ; \
+        apt -yqq install default-jre ; \
+        apt -yqq install default-jdk ; \
+        apt -yqq install curl ; \
+        apt -yqq install unzip \
+    ;
```

This syntax is pretty crazy, let's do this to avoid duplication:

```suggestion
        apt -yqq install default-jre default-jdk curl unzip ;
```

---

Done